docs/tutorials/mocks/optimistic-concurrency-control.md

```diff
@@ -592,22 +530,7 @@ rest of the `InMemoryDbCollection` methods are implemented in the `AccountManage
 which is available in the [Coyote Samples git repo](http://github.com/microsoft/coyote-samples).
 
 Build the code one last time, rewrite the assembly and run the test using Coyote for `10`
-iterations:
-
-```plain
-coyote rewrite .\AccountManager.ETags.dll
-coyote test .\AccountManager.ETags.dll -m TestGetAccountAfterConcurrentUpdate -i 10
-```
-
-Awesome, this time the test succeeds! If you try to remove the ETag check, it will fail as expected.
-
-One interesting observation is that you used a lock inside the `InMemoryDbCollection` mock but not
-inside the `AccountManager` code. You might be wondering why it was not okay to use a lock in
-`AccountManager`, but it was fine to use it in `InMemoryDbCollection`? The reason behind this choice
-is that `AccountManager` instances can run across different processes or machines in production, and
-locks do not work in such an intra-process setting. With Coyote, however, you run the entire
-concurrency unit test in a single process, so it is perfectly fine for the mock itself to take a
-lock, which makes it a lot easier to simulate the ETag functionality.
+iterations.  This time the test succeeds! If you try to remove the ETag check, it will fail as expected.
```

Comment:
Note this paragraph was removed because it was entirely redundant, an exact copy appears earlier in the document.

Reply:
Good catch!